### ISSUE TITLE
VSpacer not found

### ISSUE THREAD

**Issue by xafero** [wobsoriano/vuetify-sonner#2]
Your project seems to work, it shows my text, but that happens in the developer console:
![image](https://github.com/wobsoriano/vuetify-sonner/assets/6005006/dfcb47ea-0c9f-4e8e-a9b4-5ec144e41800)

```
{
    "dependencies": {
      "vue": "^3.2.0",
      "vuetify": "^3.0.0",
      "vuetify-sonner": "^0.2.0"
    },
    "devDependencies": {
      "@vitejs/plugin-vue": "^4.0.0",
      "typescript": "^5.0.0",
      "vite": "^4.2.0",
      "vite-plugin-vuetify": "^1.0.0",
      "vue-tsc": "^1.2.0"
    }
  }
```

```
<template>
  <v-app>
    <VSonner />
    <default-bar />
    <default-view />
  </v-app>
</template>

<script lang="ts" setup>
import { VSonner } from 'vuetify-sonner'

import DefaultBar from './AppBar.vue'
import DefaultView from './View.vue'
</script>
```

What is wrong? :-(

**Comment by wobsoriano** [comment]
Good find. Just released v0.2.1. Please try it, thanks!

**Comment by xafero** [comment]
The import works, no warning, thank you very much, especially for the fast release, too.
